models/Message.php

```diff
@@ -183,12 +183,15 @@ public static function importMessageCodes($messages, $locale = null)
 
             // Do not overwrite existing translations
             if (isset($messageData[$locale])) {
+                $item->found = 1;
+                $item->save();
```

Comment:
Is there a way we can update all existing messages with the found status in one DB query instead of one per message? That will be quite a performance impact on sites with large translation bases.

Reply:
It's only done when manually scanning the messages, but yeah, if we could somehow update with one query, that would be ideal.

Reply:
One way to improve this would be to append Message records ids into an array like:
```
if (isset($messageData[$locale])) {
   $existingIds[] = $item->id;
   continue;
}   
```
And run the following query after the loop:
```
Message::whereIn('id', $existingIds)->update(['found' => 1]);
```